<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectPage</class>
 <widget class="QWidget" name="ProjectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="8" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pushRequestToProjectButton">
     <property name="text">
      <string>Send Request to Project</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="le_request"/>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>500</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="8" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Find Project</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_search"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushSearchButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Project Data</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="l_project_name">
           <property name="text">
            <string>Project Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="l_team_name">
           <property name="text">
            <string>Team Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="l_teammates">
           <property name="text">
            <string>Teammates</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="l_post_tags">
           <property name="text">
            <string>Post Tags</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="l_project_description">
           <property name="text">
            <string>Project Description</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pushBackButton">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
